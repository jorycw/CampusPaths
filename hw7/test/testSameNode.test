# Find a path with Dijkstra's algorithm when the path 
# is to the node itself

CreateGraph graph

AddNode graph a

FindPath graph a a