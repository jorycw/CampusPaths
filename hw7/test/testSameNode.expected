# Find a path with Dijkstra's algorithm when the path
# is to the node itself

created graph graph

added node a to graph

path from a to a:
total cost: 0.000